<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IoJumpWidget</class>
 <widget class="QWidget" name="IoJumpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>680</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="checkBoxIO1">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>330</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>IO1使能</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>160</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>跳转类型</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>31</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IO2跳转:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxIO2Jump">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>71</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxIO1Jump">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>71</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>31</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IO1跳转:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxIO2">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>364</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>IO2触发模式</string>
   </property>
   <widget class="QRadioButton" name="radioButtonIO2Raise">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>上升沿</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIO2Fall">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>下降沿</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIO2High">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>高电平</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIO2Low">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>低电平</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxIO1">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>364</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>IO1触发模式</string>
   </property>
   <widget class="QRadioButton" name="radioButtonIO1Raise">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>上升沿</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIO1Fall">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>下降沿</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIO1High">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>高电平</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonIO1Low">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>低电平</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBoxIO2">
   <property name="geometry">
    <rect>
     <x>526</x>
     <y>330</y>
     <width>82</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>IO2使能</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
